libcxx/modules/CMakeLists.txt

```diff
@@ -229,6 +229,27 @@ if (LIBCXX_INSTALL_MODULES)
     )
   endforeach()
 
+  if (CMAKE_VERSION VERSION_GREATER_EQUAL "3.28.0")
```

Comment:
Note we're not using this CMake version in the CI so this will fail. But I can look at this locally.